src/main/kotlin/racingcar/Game.kt

```diff
@@ -1,19 +1,19 @@
 package racingcar
 
-class Game(private val carNumber: Int) {
-    private val carList = createCarList(carNumber)
+import java.util.Random
 
-    private fun createCarList(carNUmber: Int): List<RacingCar> {
-        val returnList = arrayListOf<RacingCar>()
-        for (number in 1..carNUmber) {
-            returnList.add(RacingCar(number.toString()))
-        }
-        return returnList
+class Game(racingCars: RacingCars) {
+    private val carList: List<RacingCar> = racingCars.carList
```

Comment:
RacingCars의 List를 꺼내서 List<RacingCar>에 할당해주고 있는데 RacingCars라는 타입 자체를 사용하는건 어떨까요?

이후 RacingCars에 우승자를 구하거나 최대값을 구하는 책임 등을 위임할 수 있을 것 같네요.

RacingCars에서 List<RacingCar>를 꺼내 사용한다면, 별도로 RacingCars라는 이름으로 래핑한 의미가 없어지는 것 같네요.

```suggestion
    private val racingCars: RacingCars = racingCars
```